src/components/@shared/FormInput/InputElement/ContainerInput/index.tsx

```diff
@@ -75,6 +75,9 @@ export default function ContainerInput(props: InputProps): ReactElement {
           name={`${field.name}[0].url`}
           checkUrl={false}
           isLoading={isLoading}
+          storageType={
+            'url' /* TODO: allow different storageType. https://github.com/oceanprotocol/market/issues/1711 */
```

Comment:
what is this? I think we should actually use this component instead of a dropdown, and then make sure we can use a lot of items beside each other. Would be way better UI than this dropdown